etcddb/etcddb_server.go

```diff
@@ -77,12 +72,9 @@ func getEtcdConf(conf *PaymentChannelStorageServerConf, cluster string) *embed.C
 		Host:   fmt.Sprintf("%s:%d", conf.Host, conf.PeerPort),
 	}
 
-	log.WithFields(log.Fields{
-		"etcd id":         conf.ID,
-		"client url":      clientURL,
-		"peer   url":      peerURL,
-		"initial cluster": cluster,
-	}).Info()
+	log.WithField("etcd server conf", fmt.Sprintf("%+v", conf)).Info()
```

Comment:
Same minor thing as in https://github.com/singnet/snet-daemon/pull/64/files#r226319411

Reply:
Updated in the branch: https://github.com/stellarspot/snet-daemon/commit/1c512e6fc2dc9be24e8904a43ecc2bf48c2532b1